### PR TITLE
Release notes 11/28 (20231112, 20231119, 20231126)

### PR DESCRIPTION
**Pull request by sabbour**
This is a placeholder. Please update the pull request when the 2023-11-28 release is out.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+* Behavioral Change
+  * Increased coredns memory limits
+  * Updated AKS add-on affinity to run on system-pool when Node Auto Provisioning is enabled
+
+* Component Updates
+  * Update Eraser version to [v1.2.3](https://github.com/eraser-dev/eraser/releases/tag/v1.2.3). This update has 3 CVE fixes:
+      * [CVE-2023-39325](https://github.com/advisories/GHSA-4374-p667-p6c8)
+      * [CVE-2023-3978](https://github.com/advisories/GHSA-2wrh-6pvc-2jm9)
+      * [CVE-2023-5043](hhttps://github.com/advisories/GHSA-qppj-fm5r-hxr3)
+  * Azure Linux image has been updated to [Azure Linux - xxxx](vhd-notes/AzureLinux/xxxx.txt).
```
Comment (sabbour) [inline CHANGELOG.md:35]: Will need to be updated with the image version on the 2023-12-03 release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-xxxx](vhd-notes/aks-ubuntu/AKSUbuntu-2204/xxxx.txt).
```
Comment (sabbour) [inline CHANGELOG.md:36]: Will need to be updated with the image version on the 2023-12-03 release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-12-03
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 is being deprecated on January 14, 2024 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy).
```
Comment (qpetraroia) [inline CHANGELOG.md:9]: Can we change this sentence to 

* Kubernetes 1.25 is being deprecated on January 14, 2024 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). Please upgrade to Kubernetes version 1.26 or above.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,40 @@
...
+
+
+* Component Updates
+  * Update Eraser version to [v1.2.3](https://github.com/eraser-dev/eraser/releases/tag/v1.2.3). This update has 3 CVE fixes:
+      * [CVE-2023-39325](https://github.com/advisories/GHSA-4374-p667-p6c8)
+      * [CVE-2023-3978](https://github.com/advisories/GHSA-2wrh-6pvc-2jm9)
+      * [CVE-2023-5043](https://github.com/advisories/GHSA-qppj-fm5r-hxr3)
+  * Azure Linux image has been updated to [Azure Linux - xxxx](vhd-notes/AzureLinux/xxxx.txt).
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-xxxx](vhd-notes/aks-ubuntu/AKSUbuntu-2204/xxxx.txt).
+
```
Comment (aritraghosh) [inline CHANGELOG.md:37]: NPD version updated to fix CVE-xxxx. Note that this change is not tied to node image upgrade, this rollout will happen outside of the regular upgrade schedule. 
@shanalily  Can you add some of the details which CVE was fixed, the version of NPD and when the rollout started

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+* [Windows containerd v1.7](https://github.com/Azure/AKS/issues/3975) will be the default container runtime for k8s v1.28+ on AKS Windows nodes. Windows Host Process (HPC) containers is GA in Windows containerd v1.7 and it has some [breaking changes](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support#container-mounts).
+* Starting Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+
+### Release notes
+
+* Preview Features
```
Comment (sabbour) [inline CHANGELOG.md:18]: Will remove `en-us` from the links.

**Comment by sabbour** [comment]
Looks good to me.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,46 @@
...
+
+
+* Component Updates
+  * Update Eraser version to [v1.2.3](https://github.com/eraser-dev/eraser/releases/tag/v1.2.3). This update has 3 CVE fixes:
+      * [CVE-2023-39325](https://github.com/advisories/GHSA-4374-p667-p6c8)
+      * [CVE-2023-3978](https://github.com/advisories/GHSA-2wrh-6pvc-2jm9)
+      * [CVE-2023-5043](https://github.com/advisories/GHSA-qppj-fm5r-hxr3)
+  * Azure Linux image has been updated to [Azure Linux - 202311.22.0](vhd-notes/AzureLinux/202311.22.0.txt).
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202312.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202312.06.0.txt).
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202312.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202312.06.0.txt).
```
Comment (palma21) [inline CHANGELOG.md:40]: 1804 is deprecated, please remove

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,46 @@
...
+* Preview Features
+  * [Artifact Streaming (Preview)](https://learn.microsoft.com/azure/aks/artifact-streaming) can now be enabled on node pools.
+  * [Cluster Autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#use-the-cluster-autoscaler-profile) parameters ignore-daemonsets-utilization, daemonset-eviction-for-empty-nodes, daemonset-eviction-for-occupied-nodes can now be configured 
+  * [Setting node soak time value](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-soak-time-value-preview) - Node soak duration helps to stagger a node upgrade in a controlled manner and minimize application downtime during an upgrade
+
+* Bug Fixes 
+  * Under some conditions it was possible to upgrade to Azure CNI Overlay from Kubenet while using the Calico network policy. This scenario is now blocked.
+
+* Behavioral Change
+  * Increased coredns memory limits.
```
Comment (palma21) [inline CHANGELOG.md:27]: From what to what?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,46 @@
...
+  * Updated AKS add-on affinity to run on system-pool when Node Auto Provisioning is enabled.
+  * Changes to reduce the kube-reserved memory reservation and eviction threshold will not be available in 1.28 as previously shared due to a release issue. These optimizations will be releasing with AKS Kubernetes minor version 1.29, which previews in January 2024.
```
Comment (palma21) [inline CHANGELOG.md:29]: Put on announcements, this is the absence of change /correction

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,46 @@
...
+  * Resource group names, Azure Virtual Network names, and subnet names for clusters with [Azure CNI Overlay networking](https://learn.microsoft.com/azure/aks/azure-cni-overlay) or [Dynamic IP allocation](https://learn.microsoft.com/azure/aks/configure-azure-cni-dynamic-ip-allocation) can't be longer than 63 characters.
```
Comment (palma21) [inline CHANGELOG.md:30]: Is this added to the docs?

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,46 @@
...
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+
+### Release notes
+
+* Preview Features
+  * [Artifact Streaming (Preview)](https://learn.microsoft.com/azure/aks/artifact-streaming) can now be enabled on node pools.
+  * [Cluster Autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#use-the-cluster-autoscaler-profile) parameters ignore-daemonsets-utilization, daemonset-eviction-for-empty-nodes, daemonset-eviction-for-occupied-nodes can now be configured 
+  * [Setting node soak time value](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#set-node-soak-time-value-preview) - Node soak duration helps to stagger a node upgrade in a controlled manner and minimize application downtime during an upgrade
```
Comment (palma21) [inline CHANGELOG.md:21]: This is GA right? If so it's feature.

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,46 @@
...
+* Starting Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+
+### Release notes
+
+* Preview Features
+  * [Artifact Streaming (Preview)](https://learn.microsoft.com/azure/aks/artifact-streaming) can now be enabled on node pools.
+  * [Cluster Autoscaler profile](https://learn.microsoft.com/azure/aks/cluster-autoscaler?tabs=azure-cli#use-the-cluster-autoscaler-profile) parameters ignore-daemonsets-utilization, daemonset-eviction-for-empty-nodes, daemonset-eviction-for-occupied-nodes can now be configured
```
Comment (palma21) [inline CHANGELOG.md:20]: Is this preview API or GA?

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,46 @@
...
+* [Windows containerd v1.7](https://github.com/Azure/AKS/issues/3975) will be the default container runtime for k8s v1.28+ on AKS Windows nodes. Windows Host Process (HPC) containers is GA in Windows containerd v1.7 and it has some [breaking changes](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support#container-mounts).
```
Comment (palma21) [inline CHANGELOG.md:12]: Good to remove, 1.28 is Ga

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (sabbour) [reply CHANGELOG.md:40]: Good to resolve this? @aritraghosh @shanalily

File: CHANGELOG.md (hunk 11 above)
Comment (sabbour) [reply CHANGELOG.md:20]: @kevinkrp93

File: CHANGELOG.md (hunk 4 above)
Comment (aritraghosh) [reply CHANGELOG.md:40]: Can we add Shoshana's comment in the bugs section

File: CHANGELOG.md (hunk 11 above)
Comment (kevinkrp93) [reply CHANGELOG.md:20]: Preview API

File: CHANGELOG.md (hunk 9 above)
Comment (kevinkrp93) [reply CHANGELOG.md:28]: https://learn.microsoft.com/azure/aks/configure-azure-cni-dynamic-ip-allocation)

File: CHANGELOG.md (hunk 7 above)
Comment (kevinkrp93) [reply CHANGELOG.md]: Removed as it is a toggle change

File: CHANGELOG.md (hunk 6 above)
Comment (kevinkrp93) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 12 above)
Comment (kevinkrp93) [reply CHANGELOG.md]: Removed

**Review by palma21 — APPROVED** [review]

**Review by qpetraroia — APPROVED** [review]

**Pull request merged**